src/Ogooreck.Sample.BusinessLogic.FSharp.Tests/Program.fs

```diff
@@ -0,0 +1,3 @@
+module Program =
```

Comment:
not required in recent times

Reply:
Good to know, cool stuff! Removed in https://github.com/oskardudycz/Ogooreck/pull/13/commits/9c3bd4fa7efd66b75bf111513ba31eea4892e0cb.